jobs/config.json

```diff
@@ -187,6 +187,13 @@
   ]
 },
 
+"ci-kubernetes-node-memcg-serial": {
+  "scenario": "kubernetes_kubelet",
+  "args": [
+    "--properties=test/e2e_node/jenkins/jenkins-memcg-serial.properties"
+  ]
+},
+
```

Comment:
I basically just want to run jenkins-memcg-serial.properties a couple times a day.